test/evergreen/code_change_report/code_change_report.py

```diff
@@ -0,0 +1,210 @@
+import argparse
+import json
+import html
+
+
+def read_code_change_info(code_change_info_path: str):
+    with open(code_change_info_path) as json_file:
+        info = json.load(json_file)
+        return info
+
+
+def get_branch_info(branches: list):
+    branch_info = list()
+    for branch in branches:
+        count = branch['count']
+        if count < 0:
+            count = 0
+        branch_info.append(count)
+    return branch_info
+
+
+def get_non_zero_count(value_list: list):
+    non_zero_count = 0
+    for value in value_list:
+        if value != 0:
+            non_zero_count += 1
+    return non_zero_count
+
+
+def get_html_colour(count: int, of: int):
+    colour = ""
+    if count == 0:
+        colour = "Red"
+    elif count == of:
+        colour = "LightGreen"
+    else:
+        colour = "Orange"
+
+    return colour
+
+
+def generate_file_info_as_html_text(file: str, file_info: dict, verbose: bool):
+    report = list()
+
+    if file.startswith("src/"):
+        report.append("<h3>File: {}</h3>\n".format(html.escape(file, quote=True)))
+
+        for hunk in file_info:
+            lines = hunk['lines']
+            report.append("<table>\n")
+            report.append("  <tr>\n")
+            report.append("    <th>Count</th>\n")
+            report.append("    <th>Branches</th>\n")
+            report.append("    <th>=+-</th>\n")
+            report.append("    <th></th>\n")
+            report.append("  </tr>\n")
+            for line in lines:
+                print("line: " + str(line))
```

Comment:
Is this a debug line, or intentionally left in?

Reply:
That was a debugging/development line, which is no longer needed. I'll delete it.